themes/jquery/js/main.js

```diff
@@ -356,6 +356,12 @@ $(function() {
 			plugin = parts[ 3 ],
 			demo = parts[ 4 ].substring( 0, parts[ 4 ].length - 5 );
 
+		var str1 = "default";
+		var str2 = "effect";
+		if ( demo.localeCompare(str1) == 0 && plugin.localeCompare(str2) == 0 ) {
```

Comment:
This is a fairly roundabout way to do this check, why not just `plugin === "effect" && demo === "default"`?